components/webui/imports/ui/SearchView/SearchResults/SearchResultsTable.jsx

```diff
@@ -0,0 +1,224 @@
+        hasMoreResults,
+        onLoadMoreResults,
+    ]);
+
+    return (
+        <div
+            id={"search-results-load-sensor"}
+            ref={loadingBlockRef}
+            style={{
+                visibility: (true === hasMoreResults) ?
+                    "visible" :
+                    "hidden",
+            }}
+        >
+            <Spinner
+                animation={"border"}
+                size={"sm"}
+                variant={"primary"}/>
+            <span>Loading...</span>
+        </div>
+    );
+};
+
+/**
+ * Represents a table component to display search results.
+ *
+ * @param {object} props
+ * @param {object} props.fieldToSortBy
+ * @param {boolean} props.hasMoreResults
+ * @param {number} props.maxLinesPerResult
+ * @param {Function} props.onLoadMoreResults
+ * @param {object} props.searchResults
+ * @param {Function} props.setFieldToSortBy
+ * @return {React.ReactElement}
+ */
+const SearchResultsTable = ({
+    fieldToSortBy,
+    hasMoreResults,
+    maxLinesPerResult,
+    onLoadMoreResults,
+    searchResults,
+    setFieldToSortBy,
+}) => {
+    const getSortIcon = (fieldName) => {
+        if (fieldName === fieldToSortBy.name) {
+            return (MONGO_SORT_ORDER.ASCENDING === fieldToSortBy.direction) ?
+                faSortUp :
+                faSortDown;
+        }
+
+        return faSort;
+    };
+
+    const toggleSortDirection = (event) => {
+        const {columnName} = event.currentTarget.dataset;
+        if (fieldToSortBy.name !== columnName) {
+            setFieldToSortBy({
+                name: columnName,
+                direction: (SEARCH_RESULTS_FIELDS.TIMESTAMP === columnName) ?
+                    MONGO_SORT_ORDER.DESCENDING :
+                    MONGO_SORT_ORDER.ASCENDING,
+            });
+        } else {
+            setFieldToSortBy({
+                name: columnName,
+                direction: (MONGO_SORT_ORDER.ASCENDING === fieldToSortBy.direction) ?
+                    MONGO_SORT_ORDER.DESCENDING :
+                    MONGO_SORT_ORDER.ASCENDING,
+            });
+        }
+    };
+
+    const rows = [];
+
+    // Construct rows
+    for (let i = 0; i < searchResults.length; ++i) {
+        const searchResult = searchResults[i];
+        rows.push(
+            <tr key={searchResult._id}>
+                <td className={"search-results-content search-results-timestamp"}>
+                    {searchResult.timestamp ?
+                        dayjs.utc(searchResult.timestamp).format(DATETIME_FORMAT_TEMPLATE) :
+                        "N/A"}
+                </td>
+                <td>
+                    <pre
+                        className={"search-results-content search-results-message"}
+                        style={{
+                            maxHeight:
+                                `${(SEARCH_RESULT_MESSAGE_LINE_HEIGHT * maxLinesPerResult)}rem`,
+                        }}
+                    >
+                        {searchResult.message}
```

Comment:
Right, the intention was to fit the timestamp into a single line. It didn't look that awkward to me the other day, but now I agree the timestamp size should match the log message size. Then we need to widen the timestamp column as well.

Another approach can be that we allow the time part of the timestamp to be wrapped into the second line, but it may not work nicely with "Max lines per result". 

What do you think?

Reply:
Yeah, I think a two-line timestamp would waste a lot of whitespace if most of the messages are a single line.

On another note, I do find the font for the messages is a bit wide. Now that I think about it, I don't think we need to use a monospace font for them, do we?

Reply:
Gotcha. Let's see if applying the non-mono version of "Rotobo" for timestamps would also help in harmonizing the styles.